docker-app/qfieldcloud/core/models.py

```diff
@@ -1764,6 +1764,27 @@ def fallback_output(self) -> str:
                 "The job ended in unknown state. Please verify the project is configured properly, try again and contact QFieldCloud support for more information."
             )
 
+    @property
+    def qgis_version(self) -> str | None:
+        """Returns QGIS app version used for the job.
+
```

Comment:
```suggestion
The QGIS version in formatted in ...
```

I think would be nice to give an example.

Reply:
```suggestion
The version number would be in `Major.Minor.Patch-NAME` format, e.g. 3.40.2-Bratislava
```

> I think would be nice to give an example.